src/glfs-cli.c

```diff
@@ -74,7 +76,8 @@ static struct cmd const cmds[] =
         { .name = "quit", .execute = handle_quit },
         { .alias = "gfrm", .name = "rm", .execute = do_rm },
         { .alias = "gfstat", .name = "stat", .execute = do_stat },
-        { .alias = "gftail", .name = "tail", .execute = do_tail }
+        { .alias = "gftail", .name = "tail", .execute = do_tail },
+        { .alias = "gflock", .name = "flock", .execute = do_flock }
```

Comment:
If `flock` is not intended to be run as a standalone program, remove the alias and that will ensure it cannot be run as a standalone program.

For example, the `connect`, `disconnect`, `help`, and `quit` commands all follow this pattern.